lib/cancan/rule.rb

```diff
@@ -140,7 +140,7 @@ def condition_match?(attribute, value)
       case value
       when Hash       then hash_condition_match?(attribute, value)
       when String     then attribute == value
-      when Enumerable then value.include?(attribute)
+      when Enumerable then enumerable_condition_match?(attribute, value)
```

Comment:
I agree, and `range.cover?` also have much better performance than `enumerable.include?`